cclib/parser/gamessdatparser.py

```diff
@@ -151,96 +156,192 @@ def extract(self, inputfile, line):
                 vectors = [float(vec) for vec in vec_line.split()[1:]]
                 line_number = line.split()[0]
 
-                if not self.mocoeffs:
-                    self.mocoeffs.append(vectors)
+                if not self.metadata['vectors']:
+                    self.metadata['vectors'].append(vectors)
 
-                elif line_number == str(len(self.mocoeffs)):
-                    self.mocoeffs[-1].extend(vectors)
+                elif line_number == str(len(self.metadata['vectors'])):
+                    self.metadata['vectors'][-1].extend(vectors)
 
                 elif len(vectors) > 0:
-                    self.mocoeffs.append(vectors)
+                    self.metadata['vectors'].append(vectors)
 
-        # Extracting Population Analysis
+        
+        # Extracting MP2 Energy Value
+
+        # MP2 NATURAL ORBITALS, E(MP2)=      -75.0022821133
+        
+        if "E(MP2)=" in line:
+            self.mpenergies = float(line.split()[-1])
```

Comment:
This should actually be a list of float (or list of list of float, can't remember) but that will become apparent once you start work on `testMP.py`.